DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsControllerTests.cs

```diff
@@ -276,6 +276,31 @@ public void AddRegistrationPromptConfigureAnswers_returns_error_with_unexpected_
             result.Should().BeRedirectToActionResult().WithControllerName("LearningSolutions").WithActionName("Error");
         }
 
+        [Test]
+        public void AddRegistrationPromptSummary_calls_custom_prompt_service_and_redirects_to_index()

```

Comment:
The title of this test implies to me that we test both that the service is called and the redirect occurs, while the test only appears to test the redirect. I think either the title should be changed to match the test, or a new assertion should be added to ensure that the service is called as expected.